gradle.properties

```diff
@@ -189,5 +189,29 @@ jfigletVersion=0.0.8
 
 embeddedRedisVersion=0.6
 
+#webjars versions
```

Comment:
Thanks for the reminder; I'd forgotten that these are also dependencies.

I do dislike the duplication, but I think the placement of the static-uris file is fine in webapps. What exactly do you envision the filtering would do? Are you thinking the filtering would take place at build-time such that the final web artifact would essentially have the versions hardcoded in the URLs as a result of that filter? That might not be so bad.